docs/architecture/cloud-native/definition.md

```diff
@@ -68,96 +69,95 @@ Cloud native systems embrace microservices. Moving beyond the popular monolithic
 
-An additional benefit is that a microservice can scale in and out independently. Instead of scaling the entire application as a single unit, you scale out specific microservices that require more processing power or network bandwidth at some specific moment. Such fine-grained scalability provides control and equates to cost savings because you need less hardware.
+An additional benefit is that each microservice can scale in and out independently. Instead of scaling the entire application as a single unit, you scale out specific microservices that require more processing power or network bandwidth. This kind of fine-grained scalability provides greater control of your system and helps to reduce overall costs as you scale portions of your system, not everything.
 
-Figure 1-4, shown below, contrasts a monolithic and microservices approach.
+Figure 1-4 contrasts a monolithic and microservices approach.
 
 ![Monolithic deployment versus microservices](media/monolithic-deployment-vs-microservices.jpg)
 **Figure 1-4.** Monolithic deployment versus microservices
 
-Cloud native systems are built as a set of independent microservices and microservices significantly increase agility and the overall maintainability of applications.
+Cloud native systems are built as a set of independent microservices that can significantly increase agility.
 
 ## Thinking in terms of containers…
 
-The use of microservices go hand-in-hand with containerization, as approach to software deployment in which a service, its dependencies, configuration ([typically, a key-value pair file](https://docs.docker.com/compose/compose-file/)) and runtime are packaged together as a single unit, called an [image](https://docs.docker.com/glossary/?term=image). Images are stored in a container registry, which can reside on your laptop, your data center or the public cloud. When needed, you can instantly transform the image into a running instance (i.e., container) and deploy it on any computer (development laptop, QA, Staging, Production, etc.) that has the Docker software engine.
+The use of microservices go hand-in-hand with containerization, an approach to software deployment in which a service, its dependencies, configuration, and runtime are packaged together as a single unit, called an [image](https://docs.docker.com/glossary/?term=image). Images are stored in a [container registry](https://azure.microsoft.com/services/container-registry/), which can reside on your laptop, your data center, or in the public cloud. When needed, you can easily *pull* the image from the registry and transform it into a running container instance on any computer (development laptop, QA, Staging, Production, etc.) that has the Docker software engine.
```

Comment:
```suggestion
The use of microservices go hand in hand with containerization, an approach to software deployment in which a service, its dependencies, configuration, and runtime are packaged together as a single unit, called an [image](https://docs.docker.com/glossary/?term=image). Images are stored in a [container registry](https://azure.microsoft.com/services/container-registry/), which can reside on your laptop, your data center, or in the public cloud. When needed, you can easily *pull* the image from the registry and transform it into a running container instance on any computer (development laptop, QA, Staging, Production, etc.) that has the Docker software engine.
```

Reply:
If you say any computer, do you really have to add the kinds of computer in parentheses?